redis-utils-lib/README.md

```diff
@@ -0,0 +1,7 @@
+# redis-utils-lib
+
+A Clojure library containing utilities for dealing with Redis.
+
+## Usage
+
+FIXME
```

Comment:
We need to put something in here to explain what this library is and how do we plan to use it.

Reply:
Yeah I was going to add that when we get a little further along into the cubby replacement. I'm not entirely sure the full extent of uses yet.